### PR TITLE
Home rules & authentication

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
Preview: https://6289d2d6c56f591d12d1cf40--celebrated-pavlova-9b844d.netlify.app/

File: src/Components/Authentication/Login.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          </div>
+          <div className="button" onClick={(e) => loginHandler(e)}>
+            Login Now
+          </div>
+          <div
+            className="button"
+            onClick={() => {
+              testlogin();
+              console.log("hello");
+              console.log("Cred:", userDetails);
```
Comment (vrx29) [inline src/Components/Authentication/Login.js:77]: you can remove console statements

File: src/Components/Authentication/Signup.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                ...userDetails,
+                confirmPassword: e.target.value,
+              })
+            }
+            type={toggleShowConfirmPass ? "password" : "text"}
+            placeholder="Confirm password"
+          ></input>
+        </div>
+        {/* <div className="policy">
+					<input value={true} data-val="true" onChange={(e) => setUserDetails({ ...userDetails, terms: !value })} type="checkbox"></input>
```
Comment (vrx29) [inline src/Components/Authentication/Signup.js:151]: you can remove commented code

File: src/Components/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <div>
+      <section className="home" id="home">
+        <div className="content">
+          <h3>Fell the freshness</h3>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. alias eos
+            facilis, ipsa est asperiores repellendus!
+          </p>
+          <button className="button btn-success" href="#down">
```
Comment (vrx29) [inline src/Components/Home/Home.jsx:24]: why are you using href on button tag>???

File: src/Components/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            {quizzesData.map((item) => {
+              return (
+                <>
+                  <FeaturedCard item={item} key={item._id} />
+                </>
+              );
+            })}
+          </div>
+        </div>
+        <div id="down"></div>
```
Comment (vrx29) [inline src/Components/Home/Home.jsx:44]: you can remove this empty div if not in use

**Review by vrx29 — APPROVED** [review]
The code is good. In few places, you have left commented code. please remove it.

File: src/Components/Header/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        <div className="icons">
+          <span className="userName">
+            hi, {authData.firstName ? authData.firstName : "User"}
+          </span>
+          <div className="fas fa-search" id="search-btn"></div>
+          {/* <div
+            onClick={toggleLightDarkTheme}
+            className={`${theme === "light" ? "fas fa-moon" : "fas fa-sun"} `}
+            id="theme-btn"
+          ></div> */}
```
Comment (vishal804) [inline src/Components/Header/NavBar.js:29]: Commented Code can be removed if not needed

File: src/Components/Rules/Rules.jsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    <main className="flex-grow-1">
+      {selectedQuizLoading && <Loader />}
+
+      {!selectedQuizLoading && (
+        <article className="rules-container ">
+          <h3 className="rules-header m-2">
+            {selectedQuizData.title} Rules, read them carefully!!!
+          </h3>
+          <div class="component-display-container">
+            <div class="grid-3-column-layout">
```
Comment (vishal804) [inline src/Components/Rules/Rules.jsx:44]: Convert Class to className

**Review by vishal804 — APPROVED** [review]
Code Looks Clean  UI Is also Good

**Pull request merged**